src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static Random random = new Random();
+    private static final int MaxNumber = 100;
+    private static final int MinNumber = 1;
```

Comment:
use constant case for constants - MAX_NUMBER

Reply:
not fixed